package.json

```diff
@@ -1,41 +1,37 @@
 {
```

Comment:
To be honest, I quite like Gulp (when it's written well) even though it's losing momentum. I much prefer it to writing bash in a package.json and it sure beats a cobbled together set of js build files. _Quietly steps back off of soapbox..._

In any case, I feel pretty strongly about keeping the format of this file readable. I mean, I see that you pinned the version of `lodash.has` which means you had to crack this guy open to remove the `^` so it's not like the `--save` was _that_ useful 😄.